Sources/Poll.php

```diff
@@ -1362,14 +1362,10 @@ protected function getMostActive(int &$options): int
 		return (int) $most_active;
 	}
 
-	/*************************
-	 * Internal static methods
-	 *************************/
-
 	/**
 	 * Validates and sanitizes $_POST input for creating or editing a poll.
 	 */
-	protected static function sanitizeInput(array &$errors): void
+	public static function sanitizeInput(array &$errors): void
```

Comment:
This is a good change, but please also move this method into the public static methods section higher up in the file.